packages/fast-usdc/src/util/noble.js

```diff
@@ -87,6 +87,10 @@ export const queryForwardingAccount = async (
   out = console,
   fetch = globalThis.fetch,
 ) => {
+  /**
+   * https://github.com/noble-assets/forwarding/blob/9d7657a/proto/noble/forwarding/v1/query.proto
+   * v2.0.0 10 Nov 2024
+   */
```

Comment:
I meant a docstring on `queryForwardingAccount`. sorry i wasn't clear. not critical.

Reply:
I would but then it makes me pull all the args into the docstring as well, quite annoying

Reply:
oh... if using `@type` rather than `@param` means no prose docstrings, that's going to be a problem real soon.